global.json

```diff
@@ -5,5 +5,10 @@
   "msbuild-sdks": {
     "MSBuild.Sdk.Extras": "3.0.44",
     "Microsoft.Build.NoTargets": "3.3.0"
+  },
+  "sdk": {
+    "version": "6.0.302",
```

Comment:
If you set this to 6.0.400-preview CI fails because CI only has GA installed which is 6.0.302

Once 6.0.400 is GA then we can bump.

Reply:
hum, how is this updated with new sdk bumps coming from maestro? we install local versions of dotnet and workloads,

Reply:
It doesn't really matter because I set
```
"rollForward": "latestMinor"
```

So dotnet is going to resolve to whatever the newest pre 7 version is.

If you pull down this PR and run
```
dotnet --version
```

It'll return 6.0.400-previewXXXX